macOS ARM64 Homebrew Install failed

Hello! In following your Homebrew installation steps for macOS arm64, an error occurs. 

```
[ 30%] Built target adm
make: *** [all] Error 2

Do not report this issue to Homebrew/brew or Homebrew/core!
```

--- a/README.md
+++ b/README.md
@@ -38,10 +38,10 @@ started.
 ## Installation
 
 ### macOS
-On macOS you can use homebrew to install the library. You just have to add the IRT’s NGA homebrew tap and can then use the usual install command.
+On macOS you can use homebrew to install the library. You just have to add the NGA homebrew tap and can then use the usual install command.
 
 ```
-brew tap irt-open-source/homebrew-nga
+brew tap ebu/homebrew-nga
 brew install libadm
 ```
 

--- a/docs/getting_started.rst
+++ b/docs/getting_started.rst
@@ -28,11 +28,11 @@ macOS
 *****
 
 On macOS you can use homebrew to install the library. You just have to
-add the IRT's NGA homebrew tap and can then use the usual install command.
+add the NGA homebrew tap and can then use the usual install command.
 
 .. code:: console
 
-    brew tap irt-open-source/homebrew-nga
+    brew tap ebu/homebrew-nga
     brew install libadm
 
 Manual installation

